apps/web/components/Organization/Wrapper/SidebarWrapper.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC, ReactNode } from 'react'
+import {
+  Box,
+  GridColumn,
+  GridContainer,
+  GridRow,
+} from '@island.is/island-ui/core'
+
+import * as styles from './SidebarWrapper.treat'
+import { useWindowSize } from 'react-use'
+import { theme } from '@island.is/island-ui/theme'
+
+interface SidebarWrapperProps {
+  sidebarContent: ReactNode
+  hideSidebarInMobile?: boolean
+  fullWidthContent?: boolean
+}
+
+export const SidebarWrapper: FC<SidebarWrapperProps> = ({
+  sidebarContent,
+  hideSidebarInMobile = false,
+  fullWidthContent = false,
+  children,
+}) => {
+  const isMobile = useWindowSize().width < theme.breakpoints.md
+
+  return (
+    <GridContainer position="none">
+      <Box
+        display="flex"
+        flexDirection={isMobile ? 'column' : 'row'}
```

Comment:
```suggestion
        flexDirection={['column', 'column', 'row']}
```

Reply:
@siggakogk Of course... thanks for the suggestion